ConsoleApp/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿internal class Program
+{
+    static int ProvideMaxValue (int number1, int number2)
+    {
+        if (number1 > number2)
+        {
+            return number1;
+        }
+        else if (number1 < number2)
+        {
+            return number2;
+        }
+        else
+        {
+            return 0;
```

Comment:
if `number1 == number2` then maxValue is either `number1` or `number2` but not zero
and not need to add `else` in any branch after first `else` because you're returning value in `if`